packages/botonic-react/src/components/multichannel/multichannel-facebook.jsx

```diff
@@ -0,0 +1,39 @@
+import { MAX_CHARACTERS_FACEBOOK } from './multichannel-utils'
+
+export class MultichannelFacebook {
```

Comment:
Move this into `multichannel/facebook/facebook.jsx`? Then we would be able to move also whatsapp to their corresponding folder.